### PR TITLE
Elmo cu 86870u6b8 google calendar   ping on discord

### PR DESCRIPTION
**Pull request by nlchung**
bot pings role listed in event description when google script discord webhook sends a msg (this occurs 15 mins before events on the Electrium General Calendar)

### CODE REVIEW

**Comment by sherwinchiu** [comment]
Task linked: [CU-86870u6b8 Google Calendar -> Ping on Discord](https://app.clickup.com/t/86870u6b8)

File: index.js [hunk 1]
```diff
@@ -3,20 +3,16 @@ const fs = require('node:fs');
 const path = require('node:path');
 const { Client, Collection, Events, GatewayIntentBits} = require('discord.js');
 const { token, channelId } = require('./config.json');
-const axios = require('axios');
```
Comment (nlchung) [inline index.js:6]: deleted unnecessary require lines to clean up index.js file

Comment (nlchung) [reply index.js:6]: just a btw in case ur wondering why i did this, feel free to resolve this once you've seen it

**Comment by nlchung** [comment]
removed review request for now bc i realized i need to fix timezone and ask sherwin abt calendar role specifics

**Review by sherwinchiu — APPROVED** [review]
lgtm

**Pull request merged**